### ISSUE TITLE
Nesting same tier Backpack?

### ISSUE THREAD

**Issue by spocane** [V0idWa1k3r/SmartBackpacks#6]
Not sure if it is intended or not, but you cannot nest the same tier backpack, 

I would really like to be able to do so, either with the normal nesting upgrade or an advanced one that is a bit more expensive and can nest any backpack in any of them.

Thanks, Loving it so far, just what i wanted!

**Comment by V0idWa1k3r** [comment]
Yeah the upgrade is missing the tooltip that you can only nest backpacks of a previous tier.
I won't be implementing this feature since it allows infinite nesting which can irreversively damage your world by exceeding the maximum depth/length for an NBT data packet and there isn't a reliable way to restrict nesting to a certain depth since there are ways to curcumvent the checks necessary and the workaround for those ways is too expensive(recursive iteration of the backpack's content) which could lead to a StackOverflow error.
TL;DR: won't be implemented due to technical limitations.
